Clarify availability of LTI 1.3 integration (#10929)
PrairieLearn/PrairieLearn: docs/lmsIntegrationInstructor.md

# LMS integration for instructors

> Integrating Learning Management Systems (LMSes) with PrairieLearn uses [LTI 1.3](https://www.1edtech.org/standards/lti) technology to connect LMS courses with PrairieLearn course instances. This allows for programmatic assignment linking and grade syncing.

Author: "passback" -> "syncing" feels slightly nicer to me.

Reviewer: Agreed.

Reviewer: Or just simplify this whole paragraph:
```suggestion
PrairieLearn can connect to Learning Management Systems (LMSes) using [LTI 1.3](https://www.1edtech.org/standards/lti) technology. This allows for assignment linking and grade syncing.
```

Author: Love it, this is much simpler and still gets the point across well.

Canvas integration via LTI 1.3 is currently available as a public preview. General availability of Canvas integration as well as support for other LMS platforms will be coming in the future.

## Prerequisites

> LTI 1.3 integration is configured on an institution level. Institution administrators will need to follow the [setup instructions](lti13.md) to establish a trust relationship between the LMS and PrairieLearn.

Author: I thought this was worth calling out, as the following steps won't be possible without this.

Reviewer: We used to have language like this but I removed it in favor of a check lower down (the paragraph starting "If PrairieLearn is not listed...").

The reason for this is that, as an instructor, I have no practical way of knowing whether this prerequisite is satisfied or not, so it feels at best irrelevant and at worst confusing. If we really do want to have language here that is directed at admins then I'd prefer to be more explicit and say something like: "Administrators should refer to the [administration LTI guide](lti13.md)." But I think that IT admins should already know that something needs to be set up?

We could also change the actionable language further down where we tell instructors what to do if PL isn't visible in their Canvas setup. We currently say to email support@PL but we could ask them to reach out to their IT people instead.

Author: Discussed with Matt, we're going to keep the existing "contact PL support" language where it already is.
